app/src/main/java/com/projectit3048c/dao/iFoodDAO.kt

```diff
@@ -5,13 +5,13 @@ import retrofit2.Call
 import retrofit2.http.GET
 import retrofit2.http.Query
 
-interface iFoodDAO {
+interface IFoodDAO {
 
-    @GET("fdc/v1/foods/search?query=egg&dataType=Foundation&pageSize=3&pageNumber=1&sortBy=dataType.keyword&sortOrder=asc&api_key=trqHcOXD3WWuC2z5OhYoMiUpZaFp2GFtEcUSs8fA")
-    fun getAllFoods() : Call<Food>
+    @GET("/anastasiiaef/JSON/food")
+    fun getAllFoods() : Call<ArrayList<Food>>
 
-    @GET("fdc/v1/foods/search?&dataType=Foundation&pageSize=3&pageNumber=1&sortBy=dataType.keyword&sortOrder=asc&api_key=trqHcOXD3WWuC2z5OhYoMiUpZaFp2GFtEcUSs8fA&")
-    fun getSearchFoods(
-        @Query("query") query: String
-    ) : Call<Food>
+   // @GET("fdc/v1/foods/search?&dataType=Foundation&pageSize=3&pageNumber=1&sortBy=dataType.keyword&sortOrder=asc&api_key=trqHcOXD3WWuC2z5OhYoMiUpZaFp2GFtEcUSs8fA&")
+    //fun getSearchFoods(
+       // @Query("query") query: String
+    //) : Call<Food>
```

Comment:
Avoid pushing commented code to Version Control.  Version control gives us better options:
- You can look at history to see what used to be there.
- You can use branches for experimental features.